Topic-Array_methods/example.js

```diff
@@ -0,0 +1,202 @@
+
+const fruits = ["Banana", "Orange", "Apple", "Mango"];
+let size = fruits.length; 
+console.log(size);
+
+/*............................................................................................................*/
+
+const fruits1 = ["Banana", "Orange", "Apple", "Mango"];
+let s=fruits1.toString();
+console.log(s);
+
+/*........................................................................................................*/
+
+const fruits2 = ["Banana", "Orange", "Apple", "Mango"];
+let fruit = fruits2.at(2);
+console.log(fruit);
+
+/*.............................................................................................................*/
+
+const fruits3 = ["Banana", "Orange", "Apple", "Mango"];
+let s2 = fruits3.join(" * ");
+console.log(s2);
+
```

Comment:
Naming conventions and declarations should be informative.

Reply:
i understood